backpack/extensions/firstorder/sum_grad_squared/sum_grad_base.py

```diff
@@ -0,0 +1,20 @@
+from backpack.extensions.firstorder.base import FirstOrderModuleExtension
+
+
+class SumGradBase(FirstOrderModuleExtension):
+    def __init__(self, derivatives, params=None):
+        self.derivatives = derivatives()
```

Comment:
Can you make this more consistent with other first-order conventions, where the base class accepts an instance of a derivative, and not the object itself ([example](https://github.com/f-dangel/backpack/blob/master/backpack/extensions/firstorder/batch_grad/batch_grad_base.py).

It should be `self.derivatives = derivatives`, and the constructor should be called in the `__init__` of a child class ([example](https://github.com/f-dangel/backpack/blob/master/backpack/extensions/firstorder/batch_grad/linear.py)).